Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.ecschema.xml

```diff
@@ -58,7 +58,7 @@
             <Class class="SlabType"/>
         </Target>
     </ECRelationshipClass>
-    <ECEntityClass typeName="Wall" displayLabel="Wall">
+    <ECEntityClass typeName="Wall" displayLabel="Wall" description="A bis:StructuralMember; two-dimensional vertical structural element used in building construction to support loads and enclose or divide spaces.">
```

Comment:
I suggest dropping the usage of semi-colons in descriptions, in favor of a more fluent single-sentence. E.g. "A bis:StructuralMember that models a vertical construction used in building construction to support loads and enclose or divide spaces"